pkg/sizeutil/size.go

```diff
@@ -0,0 +1,53 @@
+// Copyright 2017 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package sizeutil
```

Comment:
Can we name it to `types`? So that futher custom types can be placed here. Oh BTW, `timeutil.Duration` seems to fit in `types` too. :P